src/elements/content-sidebar/BoxAISidebar.tsx

```diff
@@ -70,6 +74,35 @@ const BoxAISidebar = (props: BoxAISidebarProps) => {
     } = props;
     const { questions } = cache;
     const { formatMessage } = useIntl();
+    const contextValue = React.useMemo(
```

Comment:
Are these worth memoizing? Generally useMemo should be used for expensive calculations or deeply nested objects etc.